src/app/components/utility/modal/modal.component.ts

```diff
@@ -122,9 +151,20 @@ export class ModalComponent implements OnInit {
    * Modal Confirmed.
    */
   confirmed(self) {
-    self.globalService.hideModal();
-    const PARAMS = self.globalService.formFields(self.formComponents);
-    self.confirmCallback(PARAMS);
+    if (this.editorContent === '') {
```

Comment:
```suggestion
    const content_text = document.createElement('div');
    content_text.innerHTML = this.editorContent;
    const actual_text = content_text.textContent || content_text.innerText || '';
    if (actual_text.trim() === '') {
```


So the present check doesn't handle any new lines, or blank spaces or tab spaces. Which will usually get translated to html content in frola and hence would slip through this check. 
Add the suggested code change and that should fix the issue.

Reply:
awesome!!! thanks @galipremsagar for the catch, done